cypress/e2e/user-onboarding/user-onboarding-new.ts

```diff
@@ -58,7 +62,9 @@ describe("User Onboarding (new user)", () => {
         cy.get('[role=dialog]')
             .contains("#mx_BaseDialog_title", "Download Element")
             .should("exist");
-        cy.percySnapshot("App download dialog");
+        cy.get('[role=dialog]')
+            .contains("#mx_BaseDialog_title", "Download Element")
```

Comment:
Won't this `contains` now leave the context as just the title leading to a weird screenshot?